### PR TITLE
align license with OSI template

### PR DESCRIPTION
**Pull request by lsmith77**
This will ensure that GitHub will properly detect the license.

```
licensee detect .
License:        MIT
Matched files:  LICENSE
LICENSE:
  Content hash:  4c2c763d64bbc7ef2e58b0ec6d06d90cee9755c9
  Attribution:   Copyright 2009-2018 (c) Benoît Chesneau <benoitc@e-engura.org>, 2009-2015 (c) Paul J. Davis <paul.joseph.davis@gmail.com>
  Confidence:    100.00%
  Matcher:       Licensee::Matchers::Exact
  License:       MIT
```

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -62,12 +62,6 @@ Contributing
 See `our complete contributor's guide <CONTRIBUTING.md>`_ for more details.
 
 
-License
```
Comment (apollo13) [inline README.rst:65]: I think it wouldn't hurt to keep this.

**Comment by benoitc** [comment]
from where is coming this license detection? Seems like a patch should be sent to github possibly instead of removing readability

File: README.rst (hunk 1 above)
Comment (benoitc) [reply README.rst:65]: I agree. Let just change the LICENSE file possibly. Can you provide this change?

**Comment by lsmith77** [comment]
> from where is coming this license detection? Seems like a patch should be sent to github possibly instead of removing readability. Is this possible?

you can run https://github.com/licensee/licensee locally

the tool gets confused if there are multiple references to licenses.

File: README.rst (hunk 1 above)
Comment (benoitc) [reply README.rst:65]: bump?

**Comment by benoitc** [comment]
i am not sure I like this change. This makes it barely unreadable. What I f I have multiple copyrights date. Imo this githb code should be fixed instead of asking to projects to make their list of copyrights unreadable.

**Comment by tilgovi** [comment]
Stalled. Closing.

**Pull request closed without merging**